index.html

```diff
@@ -0,0 +1,41 @@
+<!DOCTYPE html>
+<html lang="en">
+<head>
+    <meta charset="UTF-8">
+    <meta name="viewport" content="width=device-width, initial-scale=1.0">
+    <link rel="stylesheet" href="styles.css">
+    <title>Mark Portfolio</title>
+</head>
+<body>
+    <header class="header-logo">
+        <h2 class="LOGO">Mark</h2>
+        <nav>
+            <img src="images/Enabled.png" alt="" class="image-icon">
+        </nav>
+
+    </header>
+
+    <section class="landing-page all-body" id="body-cover">
+        <div class="ctn">
+             <h1 class="title">I'm Mark <br>Glad to see you!</h1>
+             <p class="intro">
+                 I'm a software developer! I can help you build a product , 
+                 feature or website Look through some of my work and experience! 
+                 If you like what you see and have a project you need coded, don't 
+                 hestiate to contact me.
+             </p>
+         
+             <p class="lets-connect">LET'S CONNECT</p>
```

Comment:
Kindly use a `<hx>` tag here instead of a `<p>` tag🙏. This is for semantic reasons as this element represents a subtitle. Remember not to use a `<h1>` tag

Reply:
Introduced <hx> tag